scipy/stats/_continuous_distns.py

```diff
@@ -9124,76 +9123,81 @@ def _rvs(self, chi, size=None, random_state=None):
         return out
 
     def _rvs_scalar(self, chi, numsamples=None, random_state=None):
-        # if chi <= 2.611:
+        # if chi <= 1.8:
         # use rejection method, see Devroye:
         # Non-Uniform Random Variate Generation, 1986, section II.3.2.
         # write: self.pdf = c * g(x) * h(x), where
         # h is [0,1]-valued and g is a density
-        # g(x) = d1 * chi**2 * x * exp(-chi**2 * (1 - x**2) / 2), 0 <= x <= 1
+        # we use two ways to write f
+        #
+        # Case 1:
+        # note that the pdf of ARGUS converges to 3*x*sqrt(1-x**2)
+        # write g(x) = 3*x*sqrt(1-x**2), h(x) = exp(-chi**2 (1-x**2) / 2)
+        #
+        # Case 2:
+        # g(x) = chi**2 * x * exp(-chi**2 * (1-x**2)/2) / (1 - exp(chi**2 /2))
```

Comment:
Where does this come from?

Reply:
![image](https://user-images.githubusercontent.com/33071866/142484214-f2be6a09-ed93-4dcc-8b47-2d802267a885.png)

I admit it is not obvious ;)

Reply:
Is this from the same source as the screenshots above or something else?